src/test/rgw/test_multi.py

```diff
@@ -468,6 +468,7 @@ def gen_secret():
 
 def gen_bucket_name():
     global num_buckets
+    global run_prefix
```

Comment:
@cbodley  Just want to make reference explicitly. your concerns make sense to me. updated.